test/Amazon.Common.DotNetCli.Tools.Test/UtilitiesTests.cs

```diff
@@ -100,5 +100,17 @@ public void TestHasExplicitSelfContainedFlag(string projectLocation, string msBu
 
             Assert.Equal(expected, result);
         }
+
+        [Theory]
+        [InlineData("TargetFramework", "", "net6.0")]
+        [InlineData("TargetFramework", "/p:NonExistence=net20.0", "net6.0")]
+        [InlineData("TargetFramework", "/p:TargetFramework=net20.0", "net20.0")]
+        public void TestPropertyEvaluationWithMSBuildParameters(string property, string msbuildparameters, string expectedValue)
```

Comment:
Worth adding a test case for when more that one property is added by the user?

Reply:
I added a couple more test scenarios.